src/config/lambda.json

```diff
@@ -24,15 +24,15 @@
         },
         {
           "key": "aws.arn",
-          "pattern": "format('arn:{}:lambda:{}:{}:function:{}', [partition, region, account_id, function_name])"
+          "pattern": "format_arn('arn:{}:lambda:{}:{}:function:{}', [partition, region, account_id, function_name])"
         },
         {
-          "key": "dt.source_entity",
-          "pattern": "dt_meid_md5('AWS_LAMBDA_FUNCTION', format('{}{}_{}', [function_name, region, account_id]))"
+          "key": "severity",
+          "pattern": "if( starts_with(log_content_parsed, '[ERROR]'), &'ERROR', &'INFO', @)"
```

Comment:
Are you parsing the log here? Is there anything to be parsed, I thougth lambda logs are a text, not json.

Reply:
I was really looking at this "parsed" thing in the name, it is addressed in other comment. I do understand the need to do this starts_with check 👍